DataScience/DataScience.py

```diff
@@ -34,6 +34,10 @@
         print("Start and end dates are expected. Example: python datascience.py 20161122 20161130")
     start_date_string = sys.argv[1]
     start_date = date(int(start_date_string[0:4]), int(start_date_string[4:6]), int(start_date_string[6:8]))
+    
+    # We will populate some data structures with info from previous day
+    start_date_prev = start_date + timedelta(days = -1)
```

Comment:
The additional amount of events we have to load depends on the experimental unit duration. For TR it’s 20h (or so). 

Maybe experimental_unit_duration = 1 
An interpret the number as days?